src/scene/scene.js

```diff
@@ -1152,6 +1153,11 @@ export default class Scene {
     // Create lighting
     createLights() {
         this.lights = {};
+
+        if (debugSettings.wireframe) {
+            Light.enabled = false; // disable lighting for wireframe mode
```

Comment:
Turns out this is a more succinct way to disable lighting for wireframe mode, rather than having to create a synthetic light.